config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,23 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func isMountpointForbidden(path string) bool {
+	// Check if the path is a restricted directory
+	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt"}
```

Comment:
Hi,

`/opt` could be added too

Reply:
Hi thanks for the feedback!
Why should /opt be added?

Reply:
makes sense thanks for letting me know!